src/core/Akka/Actor/ActorCell.cs

```diff
@@ -91,7 +91,9 @@ public ActorCell(ActorSystemImpl system, IInternalActorRef self, Props props, Me
         /// <summary>
         /// TBD
         /// </summary>
+#pragma warning disable CS0420 // A reference to a volatile field will not be treated as volatile
         public Mailbox Mailbox => Volatile.Read(ref _mailboxDoNotCallMeDirectly);
```

Comment:
We should probably remove the `Volatile.Read` call - since the field is already `volatile`.